includes/admin/onboarding/steps/step-configure_emails.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Onboarding Wizard Configure Emails Step.
+ *
+ * @package     EDD
+ * @subpackage  Onboarding
+ * @copyright   Copyright (c) 2022, Easy Digital Downloads, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       3.2
+ */
+
+namespace EDD\Onboarding\Steps\ConfigureEmails;
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+use EDD\Onboarding\Helpers;
+
+/**
+ * Initialize step.
+ *
+ * @since 3.2
+ */
+function initialize() {}
+
+/**
+ * Get step view.
+ *
+ * @since 3.2
+ */
+function step_html() {
+	$sections = array(
+		'edd_settings_emails_main' => array(
+			'email_logo',
+			'from_name',
+			'from_email',
+		),
+
+	);
+
+	$sections_purchase_receipt = array(
+		'edd_settings_emails_purchase_receipts' => array(
+			'purchase_subject',
+		),
+	);
+
+	ob_start();
+	?>
+	<form method="post" action="options.php" class="edd-settings-form">
+		<?php settings_fields( 'edd_settings' ); ?>
+		<table class="form-table" role="presentation">
+			<tbody>
+				<?php echo Helpers\settings_html( Helpers\extract_settings_fields( $sections ) ); ?>
+				<tr>
+					<th scope="row"><?php echo esc_html( __( 'Sale Notifications', 'easy-digital-downloads' ) ); ?></th>
+					<td>
+						<input type="hidden" name="edd_settings[disable_admin_notices]" value="1">
+						<label class="edd-toggle">
```

Comment:
The enabled styling on this is still not looking right to me but maybe you haven't had a chance to tweak that?